frameworks/Erlang/cowboy/src/hello_world_app.erl

```diff
@@ -24,7 +30,7 @@ start(_Type, _Args) ->
       {"/query", query_handler, []}
 		]}
 	]),
-	{ok, _} = cowboy:start_http(http, 5000, [{port, 8080}], [
```

Comment:
Presumably, these values were chosen for a specific reason. My worry here is that by lowering the number of connections available in the pool to get it working in Travis-CI, we may inadvertently degrade the performance of the actual benchmark when running on a machine that can handle that many connections.